### PR TITLE
Add image convertion script

### PR DESCRIPTION
**Pull request by brunodrugowick**

### CODE REVIEW

File: scripts/image_converter.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    fi
+}
+
+check_parameters() {
+    # Check if all mandatory parameters are provided
+    if [[ -z "$FROM" || -z "$TO" || -z "$IN" ]]; then
+        echo "Error: Missing mandatory parameters. Please provide --from, --to, and --in."
+        echo "$FROM"
+        echo "$TO"
+        echo "$IN"
```
Comment (joaoherrera) [inline scripts/image_converter.sh]: Are these `echo` intentional? If so, I suggest changing them to something like:

```bash
echo "From: $FROM"
echo "To: $TO"
echo "In: $IN
```
Check it out:

image

**Review by joaoherrera — CHANGES_REQUESTED** [review]
Nice one! Could you please provide some HEIC images so I can test you script?

File: scripts/image_converter.sh (hunk 1 above)
Comment (brunodrugowick) [reply scripts/image_converter.sh]: Oh, I shouldn't have checked these lines in.

**Comment by brunodrugowick** [comment]
> Nice one! Could you please provide some HEIC images so I can test you script?

I used these: https://github.com/tigranbs/test-heic-images/tree/master

File: scripts/image_converter.sh (hunk 1 above)
Comment (brunodrugowick) [inline scripts/image_converter.sh]:
```suggestion
        echo "Error: Missing mandatory parameters. Please provide --from, --to, and --in."
```

**Review by joaoherrera — APPROVED** [review]

**Pull request merged**